### DISCUSSION TITLE
Add filters to Boosts Information

### DISCUSSION THREAD

**Discussion by jarrettcommmm** [runelite/runelite#16174] [Ideas]
When pking, I don't need to see that all 4 skills have been brewed down, I just need to see magic level. Would be awesome if we could filter out skills tracked.

**Comment by vitorebom** [comment]
To add to this, what if we could have a setting to "group" stat boosts:

1. Group None: Current behavior, each stat boost/drain is displayed individually
2. Group Boosts: Instead of displaying each boost individually, it has a single icon to indicate "There are skills boosted"
3. Group drains: Same as the previous, but for drained stats
4. Group All: Plugin will display at most 2 icons, one to indicate that there are buffs and one to indicate that there are debuffs.  


It gets kinda funky when GWD cold/akkha shadow drains every stat and the infobox displays every single stat change.

**Comment by ShaunDreclin** [comment]
This would be really nice for leagues where you have certain skills constantly boosted +12

**Comment by AvgWdrGamer** [comment]
Yes please for the love of god! it'd be also very useful in general pvm, as boost in strength is generally the only relevant melee stat. attack/defence boosts are almost always irrelevant.
